src/main/java/moze_intel/projecte/emc/mappers/CraftTweakerRecipeMapper.java

```diff
@@ -0,0 +1,39 @@
+package moze_intel.projecte.emc.mappers;
+
+import com.google.common.collect.Iterables;
+import net.minecraft.item.crafting.IRecipe;
+import net.minecraftforge.fml.common.Loader;
+import net.minecraftforge.fml.common.ModAPIManager;
+import net.minecraftforge.fml.common.ModContainer;
+import net.minecraftforge.fml.common.versioning.VersionParser;
+
+public class CraftTweakerRecipeMapper implements CraftingMapper.IRecipeMapper {
+    private boolean ctCompat;
+
+    public CraftTweakerRecipeMapper() {
+        if (Loader.isModLoaded("crafttweaker")) { //Check to make sure it is a version of CraftTweaker that uses the new Recipe System
+            for (ModContainer mod : Iterables.concat(Loader.instance().getActiveModList(), ModAPIManager.INSTANCE.getAPIList())) {
```

Comment:
is this necessary? can't you just get the modcontainer from `Loader`?

Reply:
From what I could find by looking at the Loader class there is no method to retrieve ModContainer directly from the Loader.